enhancements/api-review/custom-tags-aws.md

```diff
@@ -91,19 +90,20 @@ userTags that are specified in the infrastructure resource will merge with userT
 
 The userTags field is intended to be set at install time and is considered immutable. Components that respect this field must only ever add tags that they retrieve from this field to cloud resources, they must never remove tags from the existing underlying cloud resource even if the tags are removed from this field(despite it being immutable).
 
-If the userTags field is changed post-install, there is no guarantee about how an in-cluster operator will respond to the change. Some operators may reconcile the change and change tags on the AWS resource. Some operators may ignore the change. However, if tags are removed from userTags, the tag will not be removed from the AWS resource.
-
-### Updated design details
+In case of hosted control plane deployments, the userTags field is updated with latest updates requested by user. The merge logic gives higher precedence to userTags when there is duplicate tag found on the AWS resource.
 
-A new field `propagateUserTags` is added in GA release version. The `experimentalPropagateUserTags` field will be deprecated. In future release versions, `experimentalPropagateUserTags` will be removed.
-When both fields are set, `experimentalPropagateUserTags` takes precedence.
+If the userTags field is changed post-install, all AWS resources created and managed by in-cluster and RedHat supported operators will be reconciled. Non-redhat supported operators may reconcile the change and change tags on the AWS resource or may ignore the change. However, if tags are removed from userTags, the tag will not be removed from the AWS resource.
 
+For the resources created and managed by hosted control plane, cluster api provider for aws reconciles the user tags on AWS resources. The hosted control plane updates the `infrastructure.config.openshift.io` resource to reflect new tags in `resourceTags`. The OpenShift operators, both core and non-core (managed by RedHat), reconcile the respective AWS resources created and managed by them. 
+Given that, there is no universal controller to update all resources created by OpenShift, the day2 updates of tags is not supported for standalone OpenShift deployments.
```

Comment:
What does it mean "not supported for standalone OpenShift deployments"? What will happen if a cluster admin changes `resourceTags` in  `infrastructure`? Line 95 ("_If the userTags field is changed post-install,_") says it's going to be applied to the cloud objects.

Reply:
At present, in standalone Openshift deployments, the `resourceTags` in `infrastructure` is handled as immutable and not a supported operation. The userTags update is allowed for hcp usecases, where, hcp control plane updates the `infrastructure` status based on edits in `hostedCluster` and `nodepool` objects.

Reply:
> resourceTags in infrastructure is handled as immutable and not a supported operation

This is not true, `status.platformStatus.aws.resourceTags` is editable on a standalone cluster.
Something *will* happen on the standalone cluster, I think the volumes just get re-tagged, just as in HCP.